js/freewheel.js

```diff
@@ -460,9 +460,9 @@ OO.Ads.manager(function(_, $) {
             slotEndedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId) {
                 amc.notifyPodEnded(adId);
               }, this, ad.id);
-            adStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(details) {
-                amc.notifyLinearAdStarted(this.name, details);
-              }, this);
+            adStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId, details) {
```

Comment:
Before, we were calling notifyLinearAdStarted with the name of the ad manager rather than the ad id. This is to fix that issue.